### ISSUE TITLE
'manimgl' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by azadatiqur** [3b1b/manim#1466]
### Code and Error
**Code**:

**Error**:

### Environment
**OS System**: 
**manim version**: master 
**python version**:

**Comment by FeronFae** [comment]
I'm having the same issue. I followed the installation video, I have all dependencies available and up to date, yet for some reason it gives me the same argument in return. I couldn't even get past the `pip install -e .` segment, it just handed me this error when I attempted to install it:
![Screenshot 2021-04-10 012059](https://user-images.githubusercontent.com/77190506/114262050-081c8700-999b-11eb-8646-40e1c3ccc779.jpg)

**Comment by azadatiqur** [comment]
I got past the `pip install -e .` successfully but still it says **"manimgl' is not recognized as an internal or external command,
operable program or batch file."**

**Comment by romanokeser** [comment]
@FeronFae did you solve it?

**Comment by FeronFae** [comment]
@romanokeser Yes, I actually did solve it. What I did was go over all the installation steps and make sure everything was installed properly. I then made sure to download the latest manimgl folder w/ manimlib from the main repository here on github and made sure I was in the right directory before executing the command. I believe the video tutorial files were somewhat outdated (? if you followed that tutorial on how to install it, that's why I came on github to follow the steps listed [here](https://github.com/3b1b/manim/blob/master/README.md). I am unsure as of what happened during my first installation attempt, but carefully repeating all the steps and making sure I wasn't missing anything seemed to do the trick. I hope it works for you too! This tool is quite amazing to make captivating animations about physics.

**Comment by thturin** [comment]
Hello, 

I am having the same issue. I am following the instruction on repo. 
"pip install -e ." installs all the packages successfully. 
When I search for installed libraries I see manimgl is installed in the "manimgl" folder. 
Everytime I try to run example_scenes.py, I get the
 _'manimgl' is not recognized as an internal or external command,
operable program or batch file._

Any suggestions?

**Comment by FeronFae** [comment]
Yes, this might be because you're referring to the Python Module as `manimgl`, yet, for some reason, unlike how it's listed on github, it actually was installed into my module library as `manimlib`. Surprisingly enough, this is how it's found as a module inside the "Manim" folder as well; try opening your "Manim" folder and check if there is a folder inside named "manimgl" or "manimlib". Perhaps try to run the command **`python -m manimlib example_scenes.py OpeningManimExample`** after the pip install, that should work. It seems that the module has changed its name pretty frequently and wasn't properly updated on the github & windows tutorials, because after installing, let me demonstrate what happens when I try to import both modules (f.ref. see attachment). Notice how importing `manimlib` returns no error, yet importing `manimgl` returns a "No module named" error:
![Screenshot 2021-05-13 155353](https://user-images.githubusercontent.com/77190506/118192477-6fe34900-b403-11eb-9521-b0bde3b04673.jpg)

**Comment by thturin** [comment]
@FeronFae  Aha I see. Thanks for this. The folder is manimlib not manimgl and when I try to import the library, it works with manimlib. Unfortunately when I run **python -m manimlib example_scenes.py OpeningManimExample**,  the code isn't executed and no manim window pops up.

**Comment by FeronFae** [comment]
Hmm, what error does it display for you when you try to run it? Perhaps I can assist you further.

**Comment by thturin** [comment]
@FeronFae sorry for the late response!!! There is no visible error when running the command via terminal. It just executes but not windows pops up from manim. When I open the project in an ide, I see these errors from  the examples_scenes.py file
![pycharm](https://user-images.githubusercontent.com/18290362/118357676-6237db80-b549-11eb-946c-ccbac62f829a.PNG)

**Comment by FeronFae** [comment]
Huh, since these are unresolved attribute errors, I would assume that perhaps the file is not reaching the rest of the python files and their respective classes + functions inside of the **manim** folder, but I'm not really sure about this issue. Sorry!

**Comment by thturin** [comment]
HI @FeronFae
Thanks for your help. I updated miktex and then ran **manimgl example_scenes.py OpeningManimExample**
despite having manimlib in my directory, manimgl still works.

**Comment by yuhan2680** [comment]
> I'm having the same issue. I followed the installation video, I have all dependencies available and up to date, yet for some reason it gives me the same argument in return. I couldn't even get past the `pip install -e .` segment, it just handed me this error when I attempted to install it: ![Screenshot 2021-04-10 012059](https://user-images.githubusercontent.com/77190506/114262050-081c8700-999b-11eb-8646-40e1c3ccc779.jpg)

Please check here is FFmpeg and Latex on your computer ,and check the python edition is between 3.7 to 3.9.
Here is another issue about this: https://github.com/3b1b/manim/issues/1730

And here is some website for you to check:
https://docs.manim.community/en/stable/installation.html
https://docs.manim.org.cn/getting_started/installation.html
http://3b1b.github.io/manim/getting_started/installation.html
